Unable to install 7.0.1 version on 16.12.0 Node

Still getting the error when trying to install 7.0.1 version on v16 Node

npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: superagent@7.0.1
npm ERR! notsup Not compatible with your version of node/npm: superagent@7.0.1
npm ERR! notsup Required: {"node":">=6.4.0 !13"}
npm ERR! notsup Actual:   {"npm":"8.3.0","node":"v16.12.0"}

same here

`
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: superagent@7.0.1
npm ERR! notsup Not compatible with your version of node/npm: superagent@7.0.1
npm ERR! notsup Required: {"node":">=6.4.0 !13"}
npm ERR! notsup Actual:   {"npm":"8.3.0","node":"v14.18.1"}
`

My node v: v14.18.1
My npm  v: 8.3.0

For me it's an error:

npm i --save-dev supertest

npm ERR! code ENOTSUP
npm ERR! notsup Unsupported engine for superagent@7.0.1: wanted: {"node":">=6.4.0 !13"} (current: {"node":"14.18.2","npm":"6.14.15"})
npm ERR! notsup Not compatible with your version of node/npm: superagent@7.0.1
npm ERR! notsup Not compatible with your version of node/npm: superagent@7.0.1
npm ERR! notsup Required: {"node":">=6.4.0 !13"}
npm ERR! notsup Actual:   {"npm":"6.14.15","node":"14.18.2"}

--- a/package.json
+++ b/package.json
@@ -75,7 +75,7 @@
     "zuul": "^3.12.0"
   },
   "engines": {
-    "node": ">=6.4.0 !13"
+    "node": ">=6.4.0 <13 || >=14"
   },
   "homepage": "https://github.com/visionmedia/superagent",
   "husky": {

